README.md

```diff
@@ -31,7 +31,15 @@ As a convenience, otto - and, by extension, Sync Gateway - does support the [Und
 Once synctos is installed, you can run it from your project's directory as follows:
 
 ```
-node_modules/synctos/make-sync-function /path/to/my-document-definitions.js /path/to/my-generated-sync-function.js
+node_modules/.bin/synctos /path/to/my-document-definitions.js /path/to/my-generated-sync-function.js
```

Comment:
This is made possible by the inclusion of a [bin](https://docs.npmjs.com/files/package.json#bin) entry in `package.json`.